spark-3.1-spanner-lib/pom.xml

```diff
@@ -42,5 +42,11 @@
       <version>4.13.2</version>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.13.3</version>
```

Comment:
This Jackson version suffers from two [CVE vulnerabilities](https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind/2.13.3). Please upgrade to version 2.15.2

Reply:
Thank you @davidrabinowitz 👏🏿👏🏿🤘🏾